### PR TITLE
Define moderation types

### PR DESCRIPTION
**Pull request by becarpenter**
Addresses https://github.com/larseggert/moderation/issues/47

### CODE REVIEW

File: draft-ecahc-moderation.md [hunk 1]
```diff
@@ -121,6 +121,14 @@ has, mostly for historic reasons, a team of moderators that are not
 list administrators and operate by a different set of processes
 {{?RFC9245}}.
 
+Note that the term "moderation" can refer both to *preemptive*
+moderation, where a moderator reviews messages before release
+to the mailing list, and *reactive* moderation, where the moderator
+intervenes after a message has been sent. The IETF mainly practices
```
Comment (larseggert) [inline draft-ecahc-moderation.md:127]: This is written for mailing lists, but the new moderation approach is supposed to cover all types of participation channels.

File: draft-ecahc-moderation.md [hunk 2]
```diff
@@ -121,6 +121,14 @@ has, mostly for historic reasons, a team of moderators that are not
...
+reactive moderation, with a spectrum from gentle reminders on- and
+off-list, all the way to suspension of posting rights and other ways
+of participating or communicating.
```
Comment (larseggert) [inline draft-ecahc-moderation.md:130]: This sounds like we want to retain sticking to mostly reactive moderation? I think we want to leave this up to the moderation team to deicide as part of their process definition.

**Review by larseggert — CHANGES_REQUESTED** [review]

**Comment by becarpenter** [comment]
Yes, update my text as you wish (can't use git from this device)
(via tiny screen & keyboard)
Regards,
        Brian Carpenter

**Comment by larseggert** [comment]
@becarpenter proposed some changes.

**Review by larseggert — DISMISSED** [review]

**Review by becarpenter — COMMENTED** [review]
All good for me, except typo s/eam/team/ on line 131

**Pull request merged**